### DISCUSSION TITLE
TUYA XR3 supported?

### DISCUSSION THREAD

**Discussion by RSV-Tuono** [arendst/Tasmota#12155] [Q&A] [answered]
Dear all,
is it possible to flash a WiFi window sensor XR3 with TASMOTA?
Bougth the same sensor 2years ago with TYWE and flashed it successfully.

What I read is that XR4 or other newer versions are not supported ...

 [Datasheet to XR3](https://developer.tuya.com/en/docs/iot/xr3-datasheet?id=K98s9168qi49g)

![WiFi Door Window Sensor](https://user-images.githubusercontent.com/28931337/119221116-14454980-baee-11eb-9dd0-92e1b47494d5.jpg)

**Comment by sfromis** [comment, marked as answer]
"XR3 is embedded with an Arm Cortex-M4F CPU" means no. You might be able to replace the XR3 with something like ESP-12, but that may not work out for battery life.

**Reply by RSV-Tuono** [reply]
OK, thank you for the information!
